### ISSUE TITLE
upgrade to resteasy 6

### ISSUE THREAD

**Issue by pjfanning** [FasterXML/jackson-jakarta-rs-providers#19]
resteasy 6 actually supports jakarta-rs while resteasy 4 is for jaxrs but it looks like some changes need to be made to this lib to support resteasy 6 (test failures)

https://mvnrepository.com/artifact/org.jboss.resteasy/resteasy-jackson2-provider

**Comment by cowtowncoder** [comment]
Yeah it'd be good to upgrade -- I had been wondering where dep to old JAXB stuff came from. But unfortunately test setup is rather fragile so simple plug-n-play won't work. I hope someone has time to figure it out, JAX-RS/Jakarta-RS providers are getting bit out of date (would be great if someone from RESTeasy et al project could help actually).
